### PR TITLE
Add Ready100 Keyboard

### PR DESCRIPTION
**Pull request by mrnoisytiger**
## Description

Add Ready100 Keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/fjlabs/ready100/rules.mk [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+KEYBOARD_LOCK_ENABLE = yes
+
+LAYOUTS = 64key
```
Comment (drashna) [inline keyboards/fjlabs/ready100/rules.mk:28]:
```suggestion
```

**Review by drashna — APPROVED** [review]

**Comment by mrnoisytiger** [comment]
Bump. 😅

File: keyboards/fjlabs/ready100/readme.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+* Kailh Hotswap sockets
+* QMK & VIA compatibility
+* Per-Key RGB LED's
+
+---
+
+* Keyboard Maintainer: FJLabs
+* Hardware Supported: Ready100
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/fjlabs/ready100/readme.md:15]: Could you add instructions on how to enter the bootloader?

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by mrnoisytiger** [comment]
Can I get review on this? Thanks!

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**